README.md

```diff
@@ -1 +1,34 @@
-# hw01
+# Homework 1: Emily Webster Biography
+
+### tl;dr Summary
+
+* Sixth-year PhD Candidate in History, working on 19th century epidemics
+* MSc student in Public Health Science
+* Lots of international travel for archival research
+* Dog and cat mom
+* Taking CFSS to beef up computing skills for epidemiological analysis/mapping
+
+### Actual Summary
+
+Hello! My name is Emily, and I am PhD Candidate in the history department here at UChicago. I work on historical epidemics of microbial disease (specifically plague, tuberculosis, and typhoid fever) across the British Empire. Because I am such a disease nerd, I am also working on an MSc in Public Health Science, so I can get a handle on epidemiological modeling and analysis. 
+
+I just returned from 18 months of dissertation research in the UK, India, and Australia, which led to some very cool moments, like these:
+
+![](./Great_Barrier_Reef.png) 
+
+*Snorkeling at the Great Barrier Reef*
+
+
+
+
+![](./Jaipur.png)
+
+*Visiting Birla Mandir in Jaipur*
+
+
+Outside of work, I love to hike and hang out with my dog and cat. I also edit a feminist environmental history blog called [Environmental History Now](https://envhistnow.com/). We try to showcase the work of female-identifying early career scholars across environmental studies. It was started by a BU student named Elizabeth Hameeteman, and has grown into something pretty remarkable - I recommend checking it out! 
```

Comment:
Nice inclusion of a link. Overall this looks very professional and informative.

Reply:
Hi Emily. I love the detail of your intro and how you used pictures and a link to know more about you. I also think it was clever to use the format of tl;dr to incorporate markdown. I couldn't see the pictures but perhaps I'm not viewing the right version.